build.gradle.kts

```diff
@@ -16,177 +20,195 @@
  */
 
 plugins {
-  id 'java'
-  id 'maven'
-  id 'java-gradle-plugin'
-  id 'net.researchgate.release' version '2.6.0'
-  id 'com.github.sherter.google-java-format' version '0.8'
-  id 'checkstyle'
-  id 'jacoco'
+  id("java")
+  id("maven")
+  id("java-gradle-plugin")
+  id("net.researchgate.release") version "2.6.0"
+  id("com.github.sherter.google-java-format") version "0.8"
+  id("checkstyle")
+  id("jacoco")
 }
 
 repositories {
   mavenCentral()
 }
 
-sourceCompatibility = JavaVersion.VERSION_1_8
-targetCompatibility = JavaVersion.VERSION_1_8
+java {
+  sourceCompatibility = JavaVersion.VERSION_1_8
+  targetCompatibility = JavaVersion.VERSION_1_8
+}
 
-group = 'com.google.cloud.tools'
+group = "com.google.cloud.tools"
 
 dependencies {
-  compile localGroovy()
-  compile gradleApi()
-  compile 'com.google.cloud.tools:appengine-plugins-core:0.9.9'
+  compile(localGroovy())
+  compile(gradleApi())
+  compile("com.google.cloud.tools:appengine-plugins-core:0.9.9")
 
-  testCompile 'commons-io:commons-io:2.4'
-  testCompile 'junit:junit:4.12'
-  testCompile 'org.hamcrest:hamcrest-library:1.3'
-  testCompile 'org.mockito:mockito-core:2.23.4'
+  testCompile("commons-io:commons-io:2.4")
+  testCompile("junit:junit:4.12")
+  testCompile("org.hamcrest:hamcrest-library:1.3")
+  testCompile("org.mockito:mockito-core:2.23.4")
 }
 
-wrapper {
-  gradleVersion = '4.9'
+
+tasks.wrapper {
+  gradleVersion = "6.9"
 }
 
-jar {
+tasks.jar.configure {
   manifest {
-    attributes 'Implementation-Title': project.name,
-        'Implementation-Version': version,
-        'Built-By': System.getProperty('user.name'),
-        'Built-Date': new Date(),
-        'Built-JDK': System.getProperty('java.version'),
-        'Built-Gradle': gradle.gradleVersion
+    attributes(
+      mapOf(
+        "Implementation-Title" to project.name,
+        "Implementation-Version" to version,
```

Comment:
This should have been `project.version` rather than `version` from a jar task (there's a `version` property in `jar` task)

Reply:
I'll add it to #437; thanks!